### PR TITLE
RPC error handling refactor

### PR DESCRIPTION
**Pull request by dubzn**
# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

# What is the current behavior?

The current behavior for various RCP methods is to perform an unwrap() which can result in a possible crash of the application.

Issue Number: #397 

# What is the new behavior?

- Removed unwrap() for RPC methods
- Custom message for InvalidCallData type errors
- The error coming from the starknet_client is taken and mapped to the corresponding BeerusErrorApi. (I would like to validate this, that's why I made [this separate PR #402](https://github.com/keep-starknet-strange/beerus/pull/402))

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

# Other information

### CODE REVIEW

**Comment by glihm** [comment]
As mentioned in #402 this PR will integrate the @dubzn changes from #402 and will focus on Starknet endpoints only.

**Comment by phklive** [comment]
Hey @dubzn how is this pr going, let's close it asap.

File: crates/beerus-rpc/src/api.rs [hunk 1]
```diff
@@ -16,6 +16,17 @@ use starknet::{
         Transaction,
     },
 };
+pub const INVALID_CALL_DATA: &str = "Invalid call data";
```
Comment (phklive) [inline crates/beerus-rpc/src/api.rs:19]: Let's add these constants in a separate file

**Review by phklive — COMMENTED** [review]
Some modifications are needed

**Comment by dubzn** [comment]
I am going to close this PR because we decided to move forward with another proposal, which is the one found in this PR: https://github.com/keep-starknet-strange/beerus/pull/490.

**Pull request closed without merging**